helm/ingress-controller/values.yaml

```diff
@@ -22,7 +22,7 @@ podAnnotations: {}
 ## @param replicaCount The number of controllers and agents to run.
 ## A minimum of 2 is recommended in production for HA.
 ##
-replicaCount: 2
+replicaCount: 1
```

Comment:
a side affect of this is that locally now we are running 1 container. I can override this easily in the makefile but i was curious if others had opinions first. On one hand, running a single container locally is simpler and faster to debug and work with, but on the other hand, it doesn't exercise our leader election important codepaths and some things may work in the context of a single agent but not multiple.